docs/src/main/asciidoc/vault.adoc

```diff
@@ -497,6 +497,27 @@ You should see:
 {password=connor}
 ----
 
+== Vault Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-vault` can add a readiness health check
+to validate the connection to the Vault server. This is disabled by default.
+
+If enabled, when you access the `/health/ready` endpoint of your application you will have information about the connection validation status.
+
+This behavior can be enabled by setting the `quarkus.vault.health.enabled` property to `true` in your `application.properties`.
+
+Only if Vault is initialized, unsealed and active, the health endpoint returns that Vault is ready to serve requests.
+
+You can change a bit this behaviour by using `quarkus.vault.health.standbyok` and `quarkus.vault.health.perfstandbyok` to `true` in your `application.properties`.
+
+standbyok:: Specifies if being a standby should still return the active status code instead of the standby status code.
+
+perfstandbyok:: Specifies if being a performance standby should still return the active status code instead of the performance standby status code.
```

Comment:
```suggestion
performance-stand-by-ok:: Specifies if being a performance standby should still return the active status code instead of the performance standby status code.
```
same here.

Reply:
I understand your point and I thought the same but in Vault, the property is named in this way `perfstandbyok` so  I decided to leave it in the same name as in Vault so anyone familiar with Vault properties finds the same property. Do you think it has sense or better to adopt to the Quarkus way?

Reply:
@lordofthejars It should be fine IMHO, Quarkus users who know about Vault will associate `performance-stand-by-ok` with a lower level Vault `perfstandbyok` without any problems :-).

Lets tweak the properties across the PR and I believe Guillaume should be fine with the PR going in

Reply:
@lordofthejars FYI, I'm holding merging this PR, #8143, so as to not affect this PR (extra rebase)